lib/hsa/mcwamp_hsa.cpp

```diff
@@ -2686,7 +2686,12 @@ class HSADevice final : public KalmarDevice
             case hc::EF_AMDGPU_MACH_AMDGCN_GFX906 : triple.append("906"); break;
         }
 
-        const auto isa{get_isa_name_from_triple(std::move(triple))};
+        unsigned sram_ecc_enabled = reader.get_flags() & hc::EF_AMDGPU_SRAM_ECC;
+        unsigned xnack_enabled = reader.get_flags() & hc::EF_AMDGPU_XNACK;
+        if (sram_ecc_enabled) triple.append("+sram-ecc");
```

Comment:
I believe the order is flipped here, XNACK comes before ECC, unless ROCr decided to be special about it.

Reply:
yup.  rocr currently has xnack first.